cookbooks/continental_extension/continental_extension.prm

```diff
@@ -7,92 +7,144 @@
+#      network evolution"
+#   5. Sandiford, D., Brune, S., Glerum, A., Naliboff, J., and Whittaker, J.M. (2021), Geophys.
+#      Geochem. Geosys., 22, e2021GC009681, "Kinematics of footwall exhumation at oceanic
+#      detachment faults: Solid-block rotation and apparent unbending"
 
 ####  Global parameters
 set Dimension                              = 2
 set Start time                             = 0
 set End time                               = 5e6
 set Use years in output instead of seconds = true
-set Nonlinear solver scheme                = single Advection, iterated Stokes
+set Nonlinear solver scheme                = single Advection, iterated defect correction Stokes
 set Nonlinear solver tolerance             = 1e-4
-set Max nonlinear iterations               = 10
+set Max nonlinear iterations               = 100
 set CFL number                             = 0.5
+set Maximum time step                      = 20e3
 set Output directory                       = output-continental_extension
 set Timing output frequency                = 1
 set Pressure normalization                 = no
 
 #### Parameters describing the model
 
-# Model geometry (400x100 km, 2 km spacing)
+# Governing equations
+subsection Formulation
+  set Formulation = Boussinesq approximation
+end
+
+# Model geometry (200x100 km, 20 km spacing)
 subsection Geometry model
   set Model name = box
   subsection Box
-    set X repetitions = 200
-    set Y repetitions = 50
-    set X extent      = 400e3
+    set X repetitions = 10
+    set Y repetitions = 5
+    set X extent      = 200e3
     set Y extent      = 100e3
   end
 end
 
-# Mesh refinement specifications (no mesh refinement,
-# but the coarse mesh is already 200x50, see above)
+# Adaptively refine the mesh to 1.25 km spacing above y=50 km
+# and between x=40 and x=160 km. These values ensure areas
+# undergoing brittle deformation are in the high-resolution
+# region
 subsection Mesh refinement
-  set Initial adaptive refinement        = 0
-  set Initial global refinement          = 0
+  set Initial adaptive refinement        = 1
+  set Initial global refinement          = 2
   set Time steps between mesh refinement = 0
+  set Strategy = minimum refinement function
+  subsection Minimum refinement function
+    set Coordinate system   = cartesian
+    set Variable names      = x,y
+    set Function expression = if ( y>=50e3 && x>=40.e3 && x<=160.e3, 4, 3)
+  end
+  set Run postprocessors on initial refinement = false
+end
+
+# Use the Eisenstat Walker method to automatically determine the
+# linear solver tolerance during the defect Picard iterations.
+# Adjusting the Maximum linear solver tolerance will affect how long
+# it takes to reach a solution, but not the actual value of the
+# solution.
+subsection Solver parameters
+  subsection Stokes solver parameters
+    set Number of cheap Stokes solver steps = 0
+  end
+  subsection Newton solver parameters
+    set Max pre-Newton nonlinear iterations      = 100
```

Comment:
That's a lot of pre-Newton steps. In essence, you're saying that Newton doesn't work for this example. Is this the case?

Reply:
Ah, thanks for catching that @bangerth! I previously used this parameter and `Nonlinear Newton solver switch tolerance` when  `iterated defect correction Stokes` was not available as a solver option and I wanted to make sure Newton iterations did not start. Removing both of these parameters.